src/app/review-rating/review-rating.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { RequestService } from '@shared/request/request.service';
+
+const api = {
+  post: {
+    reviewRating: 'api/v2/observations/review_rating/create.json',  
+  }
+};
+
+export interface RatingData {
```

Comment:
Just a suggestion. We can just name it as `Rating`/`ReviewRating`, same as how we name other things like `Activity`, `Assessment`